### ISSUE TITLE
linkosalo06

### ISSUE THREAD

**Issue by dbuona** [lizzieinvancouver/ospree#223]
I think our understanding of forcing in this exp is wrong.  We say force temp 15 for ever row, but its a bit more complicated.

The authors differentiate between forcing and warming treatments, and both involve some sort of ramping or fluctuating temperature regime.

Our figure (Fig3) reports differences in bb for treatments where temperatures were raised for different parts of the day.

I'm not sure how to proceed but I think this could be treated like a ramping study. maybe someone else can have a look?

**Comment by lizzieinvancouver** [comment]
@dbuona Did you notice the other.treatment column (below) ... any guess on what that refers to?

0.75
--
1.5
3
4
5

**Comment by dbuona** [comment]
yes, I think the forcing temperatures were applied for that many hours a day.

**Comment by lizzieinvancouver** [comment]
@dbuona Double-check then add to that cleaning clean_ambientforcing... then bounce to @lizzieinvancouver to check how I handle this other.treatment.

**Comment by dbuona** [comment]
@lizzieinvancouver, additionally,

1. The response variable is wrong. The figure (fig 3) give date of budburst, but even as far back as ospree.csv, it was already converted to Day Of Year (calculated this from 01-01-2004) but the forcing conditions started either on Feb 22, or March 8. Why is this ambiguous you might ask... 

This paper differentiates forcing and warning conditions.  **Forcing conditions** (7 day, 0 night with a temporal ramp corresponding to day length) may or may not begin at a different time than **warming** conditions (15 degrees for varying durations). Table 1k the timescale of the treatments, contradict what is stated in the methods section. 
But basically, all individuals receive (7 day and 0 night) and then a warming treatment  of 15 C is applied (for either 0.75, 1.5,3,4 or 5 hours) and this is the treatment that results in differences in day of burst.

Other problems:
photoperiod is ramping, and the duration of the **forcing** treatment changing according
**warming** is also ramped t 19 C after 4 weeks

Below I am stating my interpretation of the start conditions, if you don't mind taking a look at the paper and telling me if you agree, that would be most appreciated.

Begin counting days to budburst from February 22 2004-- I will clean in clean_misc.R.

I think the major other task is to decide how to handle the other.treatment.column, which is again, the main treatment that is influencing the results.

**Comment by lizzieinvancouver** [comment]
@AileneKane Would you mind working with Dan on this? I am a little underwater on handling so many of the papers. I can provide a third opinion if needed.

**Comment by AileneKane** [comment]
@dbuona @lizzieinvancouver I will take a look at this!

**Comment by AileneKane** [comment]
@dbuona Your interpretation looks correct to me! I the place to incorporate the other.treatment.column is in bb_daily_dataprep.

**Comment by AileneKane** [comment]
@dbuona i noticed a problem with your code in clean_misc.R. The library for for yday (is it lubridate?) needs to be loaded.

**Comment by lizzieinvancouver** [comment]
@dbuona You may be able to just use the base commands in R instead of needing to add a package. Check out conversion stuff here https://www.statmethods.net/input/dates.html
